src/extensions.jl

```diff
@@ -1,6 +1,3 @@
-lower(x::Dict{Symbol}) = BSONDict(x)
-lower(x::BSONDict) = BSONDict(x)
-
```

Comment:
This caused the bug seen in #84.

Reply:
Yeah, its a bit of a hack, and not a very good one either.